guides/source/developers/preferences/class-extension-points.html.md

```diff
@@ -51,7 +51,7 @@ module MyStore
     private
 
     def get_base_scope
-      super.where("name LIKE '%rails%'")
+      super.where("name LIKE '%Rails%'")
```

Comment:
Are my sandbox seeds not loading all products? I only see "Ruby" products, and not any "Rails" products. If the seeds only populate "Ruby" products perhaps we should change this query to `%Ruby%`.

Reply:
Looks like the sandbox only has "Ruby" products  https://github.com/solidusio/solidus/blob/master/sample/db/samples/products.rb while the

Reply:
Looks like RoR products we're recently removed: https://github.com/solidusio/solidus/commit/8221dcb5a462cf25a8117dcc326cf2db148bf862 I've created a PR to remove the old sample images: https://github.com/solidusio/solidus/pull/3397